Src/IronPython.Modules/_ctypes/_ctypes.cs

```diff
@@ -172,6 +174,15 @@ public static partial class CTypes {
             return new Win32Exception(errorCode).Message;
         }
 
+        private static string FormatError(int errorCode, string? fileName) {
+            string msg = FormatError(errorCode);
+            // error codes: https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes
+            if (errorCode is not (< 0 or >= 8200 or 34 or 106 or 317 or 718)) {
```

Comment:
How'd you pick these error codes for formatting?

Reply:
The link in the comment shows all defined error codes and their messages, grouped by ranges. I searched each range for % to see what variable fields are being used. Luckily, % placeholders are not often used. If a filename is used, it is always on %1 with the four exceptions listed. Above code 8200, filename is not used at all and %1 means something else.